lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -1105,9 +1105,9 @@ public function getDateTimeTzFormatString()
     /**
      * {@inheritDoc}
      */
-    public function getEmptyIdentityInsertSQL($quotedTableName, $quotedIdentifierColumnName)
```

Comment:
The "quoted" attribute should stay as is. It's a rabbit hole but it has its meaning. I can provide more details if necessary.

Reply:
I trust you :)